src/components/App/App.js

```diff
@@ -159,6 +266,24 @@ function App() {
   }, []);
```

Comment:
```js

    document.addEventListener('keydown', closeByEsc);

    return document.addEventListener('keypress', closeByEsc);
  }, []);

```


there is a mistake:


` return document.addEventListener('keypress', closeByEsc);` 


this adds the listener again and with `keypress` but... you need to remove the listener here passing a **clean up** functon after `return`



```js

    return () => document.removeEventListener('keydown', closeByEsc);



```


and with the same event `keydown`